config/default.yaml

```diff
@@ -396,6 +396,7 @@ live:
   rtmp:
     enabled: true
     port: 1935
+    hostname: '' # Defaults to webserver.hostname
```

Comment:
Prefer to not default to other configuration value, it's more difficult to maintain et understand for admins
Just put `'localhost'`